infra/common/src/main/java/org/apache/shardingsphere/infra/config/props/ConfigurationPropertyKey.java

```diff
@@ -122,7 +122,12 @@ public enum ConfigurationPropertyKey implements TypedPropertyKey {
     /**
      * Proxy instance type.
      */
-    PROXY_INSTANCE_TYPE("proxy-instance-type", "Proxy", String.class, true);
+    PROXY_INSTANCE_TYPE("proxy-instance-type", "Proxy", String.class, true),
+    
+    /**
+     * CDC server port.
+     */
+    CDC_SERVER_PORT("cdc-server-port", "33071", int.class, true);
     
```

Comment:
Currently, reboot is required after `cdc-server-port` changed, it might be not required later

Reply:
Yes, it is possible